src/services/index.js

```diff
@@ -1,14 +1,15 @@
 import axios from "axios";
-import { getAll } from "./products";
+import { getAll, getBySearch } from "./products";
 
 const client = axios.create({
-  baseURL: "http://localhost:3000",
+  baseURL: "https://tutti-frutti-store.herokuapp.com/",
```

Comment:
Esta URL debería estar en el `.env`
```suggestion
  baseURL: process.env.BACK_URL,
```